helix-rest/src/main/java/org/apache/helix/rest/server/json/instance/StoppableCheck.java

```diff
@@ -29,9 +29,9 @@
 public class StoppableCheck {
   // Category to differentiate which step the check fails
   public enum Category {
-    HELIX_OWN_CHECK("Helix:"),
-    CUSTOM_INSTANCE_CHECK("CustomInstance:"),
-    CUSTOM_PARTITION_CHECK("CustomPartition:");
+    HELIX_OWN_CHECK("HELIX:"),
+    CUSTOM_INSTANCE_CHECK("CUSTOM_INSTANCE_HEALTH_FAILURE:"),
```

Comment:
Do you think lower case would be more consistent with other fields and easier to read? Adding a space after colon is clearer?

Reply:
I would prefer to put the ":" into the string formatting part later. The enum info should be cleaner the better.

Reply:
Yeah.. I think the best way is to have a json class for this.. But that would need more work. Maybe we could do it later in helix rest improvement.

Reply:
This require more factor work for the TODO as we discussed before. This change only helps make clear understanding of the error. Better to address in different PR.